dashboard/test/controllers/lessons_controller_test.rb

```diff
@@ -63,6 +83,10 @@ class LessonsControllerTest < ActionController::TestCase
     assert_response :ok
     assert(@response.body.include?(@script_title))
     assert(@response.body.include?(@lesson.overview))
+    assert(@response.body.include?(@unit_group.link))
+    assert(@response.body.include?(@script.link))
+    assert(@response.body.include?(@script_title))
+    assert(@response.body.include?(@unit_group_title))
```

Comment:
these are good additions, but based on the fact that the page is likely to look different for when a unit group is present or not, I think it would be best to split this test case into one case with unit group and one case without.

Reply:
Course link is no longer being used so removed these parts of the test.